.circleci/provision.yml

```diff
@@ -60,5 +115,15 @@ workflows:
         - << pipeline.trigger_parameters.webhook.body >>
     jobs:
       - configure-cci:
+          context: 
+            - centralized-asset-github-envs
+
+  provision-infrastructure-via-circleci:
+    when:
+      and:
+        - equal: [ api, << pipeline.trigger_source >> ]
+        - << pipeline.parameters.target_repo >>
```

Comment:
Not sure what this when logic does. The API makes sense, but for target_repo, I assume it means its not empty/null? 

If so, is there a reason not to have the other parameter?